### PR TITLE
a4 added

### PR DESCRIPTION
**Pull request by shweta432**
yes

### CODE REVIEW

File: applications/a4.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
+# Afloat Tax Infrastructure Polkadot Integration
+
+- **Team Name:** Afloat Inc.
```
Comment (Shweta2217) [inline applications/a4.md:3]: teams name is not good

File: applications/a4.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+- **Payment Address:** bc1q0aghk8qufzwpmrp5nfyu9r7dh3yynmphk7rhjj
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Overview
+
+[Afloat](https://stayafloat.io/#/) is one of the first real-use cases of blockchain technology in the accounting industry. It enables the fractional buying and selling of tax credits that historically have been inefficient, opaque, and centralized. It has already processed tax credits ranging in orders from $2K -$70k USD.
+
+Afloat was built on a private Ethereum clone but wants to migrate to Polkadot due to its technology, identity management, and community. Parachains like Kilt and their identity services would be crucial at validating government roles and professional certifications like accountants and institutional sellers.
+
+## Project Details
```
Comment (Shweta2217) [inline applications/a4.md:13]: Project details is not spwcified

File: applications/a4.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+- Review and compensate appraiser
+
+Web client for:
+
+- NFT creator to request judgment and grant access to Asset Grader(s)
+- Asset Grader to review materials and provide judgment
+
+#### Phase 3
+
+User scenarios:
```
Comment (Shweta2217) [inline applications/a4.md:279]: hiii

File: applications/a4.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+#### Phase 2
+
+User scenarios:
+
+- Registration/approval of new appraisers,
+- Request appraisal of NFT,
+- Appraise NFT,
+- Review and compensate appraiser
+
+Web client for:
```
Comment (Shweta2217) [inline applications/a4.md:272]: hii

File: applications/a4.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+| 0e. | Article | We will publish an **article** in English and Spanish that explains what was built and how it can benefit other projects
+| 1. | Approve Redemption Specialists | Pallet and web client for onboarding and approving/enrolling Tax Credit Redemption Specialists. These are experts that know how to perform appropriate steps to migrate a tax credit IRL. Approval will be handled by marketplace administrator initially, by the community eventually. |
+| 2. | Request Redemption | Pallet and web function/form for requesting redemption, optionally from a specific Redemption Specialist or system will choose. |
+| 3. | View materials and Redeem | Pallet support and web client for Redemption Specialist to review the encrypted tax credit materials and confirm that off-chain IRL redemption has been completed. Redemption completion will be approved by owner and Redemption specialist. |
+| 4. | Asset Manager | Once part or all of a tax credit NFT is redeemed, the Tax Credit Asset Manager(s) will be able to lock, freeze, and/or burn the NFT. |
+| 5. | Launch Materials | Launch materials, videos and speaking arrangements for Louise are already in queue.|  
+
+## Future Plans
```
Comment (Shweta2217) [inline applications/a4.md:253]: hiii

**Comment by Shweta2217** [comment]
ywes

**Comment by shweta432** [comment]
noooo

**Review by shweta432 — COMMENTED** [review]
from 432

**Comment by Shweta2217** [comment]
new comment

File: applications/a4.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+
+[Hypha DAO](https://dho.hypha.earth/#/): Smart contracts and front end development that enables the creation of flexible roles, assignments and contributions with recurring payments. Design and implement a graph data layer to improve web application performance. Design and build a [Double Entry accounting](https://us02web.zoom.us/rec/share/eRqiBvq-dsV0L_hEjW5e8DWNYQlUn2bLhI8-86jkRVwdXiN3TiD5edym17ubCd9R.QhKQw_Byy0t5_8SW?startTime=1647371674000) (Passcode: .V$C#Br2) plattform that streams wallet activity, supports token price history, reporting and currency conversion.
+
+[SEED](https://joinseeds.earth/): Smart contract and mobile development that capture the project's constitution, enable voting on proposals and basic identity management like reputation, vote history etc. Design and build a PWA token swaps app. Design and build a basic [Economic Simulator](https://seeds-sim.hypha.earth/dashboard) that enables voters to understand the economic impact of policy changes.
+
+### Relevant profile links
+
```
Comment (Shweta2217) [inline applications/a4.md:156]: hiiii

**Review by Shweta2217 — COMMENTED** [review]
nb

**Review by Shweta2217 — COMMENTED** [review]
n bb b

**Review by Shweta2217 — COMMENTED** [review]
something chnaged

**Pull request merged**